index.d.ts

```diff
@@ -29,6 +29,7 @@ declare namespace JwksRsa {
   interface CertSigningKey {
     kid: string;
     nbf: string;
+    pem: string;
```

Comment:
Still need to remove pem from here and add getPublicKey no?

Reply:
Good catch